terraform-dev/main-gcp.tf

```diff
@@ -104,6 +104,10 @@ variable "alerts_error_message_no_data" {
   type = string
 }
 
+variable "enable_redis_session_store_instance" {
+  default = true
+}
```

Comment:
This works, but the convention in this repo is to define variables in `environment.auto.tfvars` if their value differs between environments.  Then reference it in this file with:
```tf
variable "enable_redis_session_store_instance" {
  type = string
}
```